lib/matplotlib/patches.py

```diff
@@ -3407,8 +3407,7 @@ def transmute(self, path, mutation_size, linewidth):
             return _path, _fillable
 
     class Curve(_Curve):
-        """
-        A simple curve without any arrow head.
+        """A simple curve without any arrow head.
```

Comment:
Are all these single-line changes really necessary? Personally, I don't think they look very good like this.